pgsql-flexible-server.tf

```diff
@@ -72,3 +91,30 @@ resource "azurerm_postgresql_flexible_server_configuration" "pgsql_server_config
   server_id = azurerm_postgresql_flexible_server.pgsql_server.id
   value     = each.value.value
 }
+
+
+resource "azurerm_postgresql_flexible_server_active_directory_administrator" "pgsql_adadmin" {
+  server_name         = azurerm_postgresql_flexible_server.pgsql_server.name
+  resource_group_name = azurerm_postgresql_flexible_server.pgsql_server.resource_group_name
+  tenant_id           = data.azurerm_client_config.current.tenant_id
+  object_id           = data.azuread_group.db_admin.object_id
+  principal_name      = local.admin_group
+  principal_type      = "Group"
+}
+
+
+resource "postgresql_role" "aad_role" {
+  name = local.admin_group
```

Comment:
This should be the JIT group not the admin group, admin group will already have access,

naming convention is here:
https://github.com/hmcts/cnp-module-postgres/blob/40306e4774dbde9a5c164cb7bd3b513e4da5cbf8/db-read-access.tf#L2

they are normally created manually outside of code currently as a one-off when the team starts up